src/foundry/client/pixi/core/interaction/mouse-handler.d.ts

```diff
@@ -156,144 +171,70 @@ declare class MouseInteractionManager<Object extends PIXI.Container = PIXI.Conta
-   * @internal
-   */
-  protected _activateClickEvents(): void;
-
-  /**
-   * Deactivate event listeners for click events on the target object
-   * @internal
-   */
-  protected _deactivateClickEvents(): void;
-
-  /**
-   * Activate events required for handling a drag-and-drop workflow
-   * @internal
-   */
-  protected _activateDragEvents(): void;
-
-  /**
-   * Deactivate events required for handling drag-and-drop workflow.
-   * @internal
-   */
-  protected _deactivateDragEvents(): void;
-
-  /**
-   * Handle mouse-over events which activate downstream listeners and do not stop propagation.
-   * @internal
-   */
-  protected _handleMouseOver(event: PIXI.InteractionEvent): unknown;
-
-  /**
-   * Handle mouse-out events which terminate hover workflows and do not stop propagation.
-   * @internal
-   */
-  protected _handleMouseOut(event: PIXI.InteractionEvent): unknown;
-
-  /**
-   * Handle mouse-down events which activate downstream listeners.
-   * Stop further propagation only if the event is allowed by either single or double-click.
-   * @internal
-   */
-  protected _handleMouseDown(event: PIXI.InteractionEvent): unknown;
-
-  /**
-   * Handle mouse-down which trigger a single left-click workflow.
-   * @internal
-   */
-  protected _handleClickLeft(event: PIXI.InteractionEvent): void;
-
-  /**
-   * Handle mouse-down which trigger a single left-click workflow.
-   * @internal
-   */
-  protected _handleClickLeft2(event: PIXI.InteractionEvent): unknown;
-
-  /**
-   * Handle right-click mouse-down events.
-   * Stop further propagation only if the event is allowed by either single or double-click.
-   * @internal
-   */
-  protected _handleRightDown(event: PIXI.InteractionEvent): unknown;
-
-  /**
-   * Handle single right-click actions.
-   * @internal
-   */
-  protected _handleClickRight(event: PIXI.InteractionEvent): void;
-
-  /**
-   * Handle double right-click actions.
-   * @internal
-   */
-  protected _handleClickRight2(event: PIXI.InteractionEvent): unknown;
-
-  /**
-   * Handle mouse movement during a drag workflow
-   * @internal
-   */
-  protected _handleMouseMove(event: PIXI.InteractionEvent): unknown;
-
-  /**
-   * Handle the beginning of a new drag start workflow, moving all controlled objects on the layer
-   * @internal
-   */
-  protected _handleDragStart(event: PIXI.InteractionEvent): unknown;
-
-  /**
-   * Handle the continuation of a drag workflow, moving all controlled objects on the layer
-   * @internal
+   * A reference to the possible interaction states which can be observed
    */
-  protected _handleDragMove(event: PIXI.InteractionEvent): unknown;
+  get handlerOutcomes(): ValueOf<MouseInteractionManager.HANDLER_OUTCOME>;
 
   /**
-   * Handle mouse up events which may optionally conclude a drag workflow
-   * @internal
+   * A public method to handle directly an event into this manager, according to its type.
+   * Note: drag events are not handled.
+   * @param event - No comment
```

Comment:
If there's truly no comment, to avoid a linting error just remove this line entirely. If Foundry says "No comment"... I'd probably still remove it in this case. It'd be confusing to keep it around.